packages/vsce/package.json

```diff
@@ -895,7 +895,6 @@
     "@zowe/core-for-zowe-sdk": "^8.0.0-next.202402021649",
     "@zowe/zowe-explorer-api": "^3.0.0-next",
     "axios": "^1.6.7",
-    "https": "~1.0.0",
```

Comment:
I believe this was added for the purpose of bundling purposes. 😋 

If this is a bit too strange, we can investigate how to move it into `bundledDependencies` without impacting anything else 😋

Reply:
Well, it doesn't really seem to do anything, which is why I removed it. Seems quite a few people are importing this package as well.

Reply:
You are right, not used 😋